### ISSUE TITLE
getBlendMode should default to using BLEND_MODES rather than blendModes

### ISSUE THREAD

**Issue by AlmirKadric** [CloudKidStudio/PixiParticles#8]
Right now ParticleUtils.getBlendMode is using `blendModes` as a default and will only fall back to `BLEND_MODES` if `blendModes` doesn't exist. As a result PIXI is throwing a deprecation warning. Hoping we can get rid of this unnecessary noise.

EDIT: After playing around a bit more, seems there are quite a lot of functions using deprecated API and doing the same as above. This hurts performance quite a bit as console.[anything kind of log] is extremely slow.

**Comment by bQvle** [comment]
Hi AlmirKadric,

Its actually just checking what method it should use, and goes with the newer once. you can easily just remove the check, just search for the deprecated functions, and remove the checks.

something like,

If (PIXI.blendModes)
 blendmode = PIXI.blendModes
else
 blenmode = PIXI.BLEND_MODES

I think this is build in to support multiple versions of Pixi-js.

**Comment by AlmirKadric** [comment]
Hi @bQvle,

I think you misunderstood what I wrote. The fact that it's trying to check which function to use is not the problem. The problem is that it doesn't use the newer one by default.

for example: the newer property for blendMode is BLEND_MODE however the current code is defaulting to blendMode. This then causes the deprecation warning in PIXI to be written to the console.

I have opened up #9 which fixes the issues i have come across so far.
